api.php

```diff
@@ -43,6 +43,20 @@ function wp_consent_api_enqueue_assets( $hook ) {
 }
 add_action( 'wp_enqueue_scripts', 'wp_consent_api_enqueue_assets', PHP_INT_MAX-100 );
 
+/**
+ * Enqueue style for back-end  to show wp-consent-api unregister plugins list in a better style in site health check page.
+ *
+ * @param $hook
+ */
+function wp_consent_api_enqueue_admin_assets($hook) {
```

Comment:
There should be a space on either side of $hook to match with WP coding standards.